src/walletConnect/v2/saga.ts

```diff
@@ -67,14 +67,14 @@ import {
 import networkConfig from 'src/web3/networkConfig'
 import { getWalletAddress } from 'src/web3/saga'
 
-let client: SignClient | null = null
+export let client: IWeb3Wallet | null = null
```

Comment:
why exporting this now?

Reply:
Fixed in ec3c8885e1dded575c1c79f62df8debba905775a!